pkg/config/orderer_test.go

```diff
@@ -796,3 +1158,94 @@ func TestRemoveOrdererEndpointFailure(t *testing.T) {
 		})
 	}
 }
+
+func baseOrdererOfType(ordererType string) Orderer {
+	switch ordererType {
+	case ConsensusTypeKafka:
+		return baseKafkaOrderer()
+	case ConsensusTypeEtcdRaft:
+		return baseEtcdRaftOrderer()
+	default:
+		return baseSoloOrderer()
+	}
+}
+
+func baseSoloOrderer() Orderer {
+	return Orderer{
+		Policies:    ordererStandardPolicies(),
+		OrdererType: ConsensusTypeSolo,
+		Organizations: []Organization{
+			{
+				Name:     "OrdererOrg",
+				Policies: orgStandardPolicies(),
+				OrdererEndpoints: []string{
+					"localhost:123",
```

Comment:
Here’s a bit of where we see how we have different ways of handling addresses. Earlier we had host string and port uint while here we have an actual address as expected by dial.

Reply:
I'm going to end up standardizing to an Address type because Anchor Peer logic already supports that and it's easier to convert in one direction than the other.